index.js

```diff
@@ -57,15 +58,16 @@ class ReactRouterGA extends React.Component<Props> {
 
     this.lastPathname = location.pathname;
 
-    // Sets the page value on the tracker.
-    window.ga('set', 'page', location.pathname);
+    // Sets the page value on the tracker. If a basename is provided, then it is prepended to the pathname.
+    const page = this.props.basename ? `${this.props.basename}${location.pathname}` : location.pathname;
```

Comment:
Can we please have an extra line b/w the constant declaration and the call to `ga`?